packages/desktop-electron/preload.js

```diff
@@ -1,4 +1,6 @@
 const { ipcRenderer, contextBridge } = require('electron');
+// TODO: remove node module usage from this file
+// see: https://www.electronjs.org/docs/latest/tutorial/sandbox
```

Comment:
This is where I got stuck. Basically we either need to flip over `nodeIntegration` from `false` to `true` (which has security implications) or we need to get rid of these dependencies (move them someplace else?).